### PR TITLE
use contents of startup_script_1 and startup_script_2.

### PR DESCRIPTION
**Pull request by bmenasha**
previously the examples would fail as the startup-script metadata was the string
"../../modules/sap_hana/files/startup.sh" rather then the actual startup script

### CODE REVIEW

File: modules/sap_hana/main.tf [hunk 1]
```diff
@@ -96,7 +96,7 @@ resource "google_compute_instance" "gcp_sap_hana" {
     sap_hana_sidadm_uid        = "${var.sap_hana_sidadm_uid}"
     sap_hana_sapsys_gid        = "${var.sap_hana_sapsys_gid}"
 
-    startup-script = "${var.startup_script}"
+    startup-script = "${file(var.startup_script)}"
```
Comment (morgante) [inline modules/sap_hana/main.tf]: Instead of breaking the module interface, could we move this into the example code? So the module itself still accepts the actual startup script (not a file path)?

**Review by morgante — CHANGES_REQUESTED** [review]

**Comment by bmenasha** [comment]
sure. done.
I also did some cleanup on the test fixtures, no need to duplicate the startup script for each test.
thanks

**Review by morgante — APPROVED** [review]
